### PR TITLE
Abstracts: rendering of submission information

### PR DESCRIPTION
**Pull request by pferreir**

### CODE REVIEW

File: indico/htdocs/sass/modules/abstracts/_abstract_page.scss [hunk 1]
```diff
@@ -141,3 +140,64 @@
...
+.submitted-data {
+    .field-separator {
+        @include border-top;
+        margin: 0 -10px;
+        padding: 1em 10px;
+    }
+
+    .comment {
+        margin-bottom: 1em;
+        @extend .text-not-important;
```
Comment (nop33) [inline indico/htdocs/sass/modules/abstracts/_abstract_page.scss]: I usually put the `@extend` and `@include` in the beginning of the selector's body and leave an empty line after. I've seen it in quite a few places. Maybe we could make it a "rule"? Just my opinion.

File: indico/htdocs/sass/modules/abstracts/_abstract_page.scss [hunk 2]
```diff
@@ -141,3 +140,64 @@
...
+    }
+
+    dl.custom-field-values {
+        .field-value {
+            margin-bottom: 0.5em;
+            & > * {
```
Comment (nop33) [inline indico/htdocs/sass/modules/abstracts/_abstract_page.scss]: Empty line before the beginning of every selector inside the body of another maybe?
```scss
dl.custom-field-values {
    .field-value {
        margin-bottom: 0.5em;
     
        & > * {
            ....
        }
    }
}
```

Comment (OmeGak) [inline indico/htdocs/sass/modules/abstracts/_abstract_page.scss]: On a different note, you don't need `&` here.

File: indico/modules/events/abstracts/blueprint.py [hunk 3]
```diff
@@ -39,6 +40,8 @@
 # Display pages
 _bp.add_url_rule('/abstracts/<int:abstract_id>/', 'display_abstract', RHDisplayAbstract)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
+_bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
```
Comment (OmeGak) [inline indico/modules/events/abstracts/blueprint.py]: Do we need `<filename>` at all?

Comment (ThiefMaster) [inline indico/modules/events/abstracts/blueprint.py]: We don't need one but it's much prettier to have one (we do the same thing for other download endpoints too)

File: indico/htdocs/sass/modules/abstracts/_abstract_page.scss [hunk 4]
```diff
@@ -141,3 +140,64 @@
...
+.submission-info {
+    .info-line {
+        margin-top: 1em;
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        .author-list {
+            font-size: 0.95em;
```
Comment (pferreir) [inline indico/htdocs/sass/modules/abstracts/_abstract_page.scss]: It is perceptible. No particular reason, just looks nice.

**Pull request merged**